Builds/containers/shared/install_cmake.sh

```diff
@@ -2,9 +2,9 @@
 set -ex
 
 cd /tmp
-CM_INSTALLER=cmake-3.13.2-Linux-x86_64.sh
-CM_VER_DIR=/opt/local/cmake-3.13
-wget https://cmake.org/files/v3.13/$CM_INSTALLER
+CM_INSTALLER=cmake-3.15.2-Linux-x86_64.sh
```

Comment:
What is our min version of cmake? Do we have a CI configuration with this min version? No change needed, but the upgrade made me wonder.

Reply:
`cmake_minimum_required` says 3.9 currently. I would personally prefer 3.12 or higher, but I agree we should have a test for whatever that version is. I'll add another TODO item for that.